front/src/components/oauth2/actions.js

```diff
@@ -0,0 +1,112 @@
+import { RequestStatus } from '../../utils/consts';
+
+const actions = store => ({
+  updateIntegrationName(state, e) {
+    store.setState({
+      integrationName: e
+    });
+  },
+  updateClientId(state, e) {
+    store.setState({
+      clientId: e.target.value
+    });
+  },
+  updateSecretId(state, e) {
+    store.setState({
+      secretId: e.target.value
+    });
+  },
+  async getCurrentConfig(state) {
+    store.setState({
+      oauth2GetStatus: RequestStatus.Getting
+    });
+
+    const returnServiceId = await state.httpClient.get(`/api/v1/service/${state.integrationName}/getServiceId`);
```

Comment:
I think `{get} /api/v1/service/${state.integrationName}` is exptected. No `/getServiceId`.

```
  /**
   * @api {get} /api/v1/service/:service_name getByName
   * @apiName getByName
   * @apiGroup Service
   * @apiSuccessExample {json} Success-Example
   * {
   *  "id":"57ae1702-c071-483a-b532-384a507c1f04",
   *  "pod_id":null,
   *  "name":"rtsp-camera",
   *  "selector":"rtsp-camera",
   *  "version":"0.1.0",
   *  "status":"RUNNING",
   *  "has_message_feature":false,
   *  "created_at":"2019-06-14T04:38:51.555Z",
   *  "updated_at":"2019-06-14T04:38:51.555Z"
   * }
   */
```

Reply:
I fix it